dbscripts/xml/upgrade/3.0.0_versioning.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE data SYSTEM "../../../lib/pkp/dtd/xmlData.dtd">
+
+<!--
+  * 3.0.0_versioning.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
```

Comment:
And I've recently changed

> Copyright (c) xxxx-yyyy Simon Fraser University Library

...to...

> Copyright (c) xxxx-yyyy Simon Fraser University

...as SFU is the legal entity, not the SFU library.